charts/k8ssandra-cluster/values.yaml

```diff
@@ -7,6 +7,9 @@ k8ssandra:
   configuration:
     auth:
       enabled: true
+      superuser:
+        secret: ""
+        username: ""
```

Comment:
Should we have a sane default here so that user doesn't always have to set it? What if they would just want to enable auth and have this set to "admin" + generated password? I think that would provide nicer "out-of-box" experience.

Reply:
>Should we have a sane default here so that user doesn't always have to set it? 

If not set, cass-operator generates a secret of the form `{clusterName}-superuser`.

Reply:
Probably something we want to documentate, otherwise the empty username feels kinda odd.